src/main/java/org/jasig/portlet/proxy/service/GenericContentRequestImpl.java

```diff
@@ -20,49 +20,56 @@
 
 import javax.portlet.PortletPreferences;
 import javax.portlet.PortletRequest;
+import javax.portlet.WindowState;
 
 /**
  * GenericContentRequestImpl represents a basic content request consisting
  * simply of a target location.
- * 
+ *
  * @author Jen Bourey, jennifer.bourey@gmail.com
  */
 public class GenericContentRequestImpl implements IContentRequest {
 
-	private String proxiedLocation;
+    private String proxiedLocation;
+
+    /**
+     * Standard key for storing a content location in the portlet preferences.
+     */
+    public static final String CONTENT_LOCATION_KEY = "location";
+    public static final String CONTENT_MAX_LOCATION_KEY = "max_location";
```

Comment:
FWIW, I normally (at least recently) name these things *_PREFERENCE instead of *_KEY (or something else) because I feel like the word PREFERENCE brings additional clarity in reading the code.  (I see that CONTENT_LOCATION_KEY was there already.)